### PR TITLE
[DD-268] Add banner atop legacy circle ci v1 api

### PR DESCRIPTION
**Pull request by ccgeller**
Jira case: [DD-268](https://circleci.atlassian.net/jira/software/c/projects/DD/boards/477?modal=detail&selectedIssue=DD-268)
Preview link: [link](http://circleci-doc-preview.s3-website-us-east-1.amazonaws.com/DD-268-Add-banner-atop-legacy-CircleCI-v1-API-preview/api/v1/#circleci-v1-api-overview)

# Description
Add banner atop legacy CircleCI v1 API docs to direct to CircleCI v2 API docs so that user use the v2 APIs

# Reasons:
Direct users to v2 APIs so the v1 APIs can eventually be deprecated

# Changes
- add in css for banner

# Before
Screen Shot 2021-12-16 at 7 57 19 PM

# After
Screen Shot 2021-12-21 at 3 40 36 PM

Banner sticky to top
Screen Shot 2021-12-21 at 3 47 54 PM

### CODE REVIEW

File: src-api/source/layouts/layout.erb [hunk 1]
```diff
@@ -106,6 +106,9 @@ under the License.
     <div class="page-wrapper">
       <div class="dark-box"></div>
       <div class="content">
+        <aside class="v2-banner notice">
```
Comment (teesloane) [inline src-api/source/layouts/layout.erb:109]: Nice use of semantic html! :)

**Review by teesloane — APPROVED** [review]
LGTM!

**Comment by nramabad** [comment]
Hey @danbeere what do you think about making the banner sticky so that it floats at the top?

It bothers me that if you navigate to a #section directly via the URL, you would miss the banner.

**Comment by danbeere** [comment]
@nramabad Making the banner sticky so that it floats at the top is a good idea. I missed it when the page loaded and navigated to the section.

**Comment by nramabad** [comment]
@ccgeller can you address the scroll-margin-top? when I navigate to a section directly, the banner covers up the section title. can you address that before I approve? ty!

**Review by nramabad — APPROVED** [review]
LGTM!

**Pull request merged**